src/index.html

```diff
@@ -1,17 +1,212 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
+    <meta 
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, 
+              user-scalable=no, 
+              initial-scale=1.0, 
+              maximum-scale=1.0, 
+              minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.
+      com/create-application" 
+      method="post" 
+      autocomplete="off"
+    >
+
+      <fieldset class="field">
+
+        <legend>Personal information</legend>
+
+        <div class="line">
+          <label for="surname">Surname:</label>
+          <input 
+            id="surname" 
+            type="text" 
+            name="surname" 
+            minlength="2" 
+            maxlength="20" 
+            required 
+            tabindex="1"
```

Comment:
```suggestion
            id="surname"
            type="text"
            name="surname"
            minlength="2"
            maxlength="20"
            required
            tabindex="1"
```
нужно убрать лишние пробелы в конце строк